services/QuillLMS/client/app/bundles/Diagnostic/components/eslDiagnostic/languagePage.jsx

```diff
@@ -9,7 +9,7 @@ export class LanguagePage extends React.Component {
   handleClickLanguage = (e) => {
     const language = e.currentTarget.value;
     const { dispatch, setLanguage, previewMode, begin } = this.props;
-    if(process.env.NODE_ENV === 'production' && language !== ENGLISH) {
+    if(language !== ENGLISH) {
```

Comment:
Good catch on the issue here.  We probably actually DO want to be setting `NODE_ENV` on our environments, so in the long run we should probably revisit this, but no point in doing a whole bunch of work around that now.